### PR TITLE
Issue 89: Reviewed configure-agent.md

### PR DESCRIPTION
**Pull request by iguitton**
* reviewed the entire file
* restructured the file
* rephrased sections for better clarity
* updated the quick start page to be consistent with the updated version of configure-agent.md

### CODE REVIEW

**Comment by iguitton** [comment]
@NassimBtk , this branch should be merged after https://github.com/sentrysoftware/metricshub/pull/90

File: metricshub-doc/src/site/markdown/configuration/configure-agent.md [hunk 1]
```diff
@@ -421,41 +396,24 @@ resourceGroups:
...
-  otel.exporter.otlp.metrics.endpoint: https://my-host:4317
-  otel.exporter.otlp.logs.endpoint: https://my-host:4317
+  otel.exporter.otlp.metrics.endpoint: https://<my-host>:4317
+  otel.exporter.otlp.logs.endpoint: https://<my-host>:4317
 
 resourceGroups: #...
 ```
 
-##### Example Configuration for the **MetricsHub Agent** Community Edition to Transmit Metrics to the Prometheus OTLP Receiver
+where `<my host>` should be replaced with the hostname or IP address of the server where the OTLP receiver is installed.
```
Comment (NassimBtk) [inline metricshub-doc/src/site/markdown/configuration/configure-agent.md:414]: Small typo, we should reference `<my-host>`

**Review by NassimBtk — CHANGES_REQUESTED** [review]
Minor updates, conflict resolution and we should be good.

**Comment by iguitton** [comment]
Thanks @NassimBtk for your feedback. I took your comments into account. Please re-review :)

**Review by NassimBtk — APPROVED** [review]

**Pull request merged**